pytorch_lightning/plugins/precision/precision_plugin.py

```diff
@@ -48,6 +50,14 @@ def connect(
         """Connects this plugin to the accelerator and the training process."""
         return model, optimizers, lr_schedulers
 
+    @property
+    def scaler(self) -> Optional["GradScaler"]:
```

Comment:
Should we raise a NotImplementedError instead? Same above ?

Reply:
The current logic in accelerator is 
https://github.com/PyTorchLightning/pytorch-lightning/blob/700521c7d353f79f38d1146cffc63bbe23df8e66/pytorch_lightning/accelerators/accelerator.py#L267-L273
and what pasted above. 
I think it mean to be None base on the current logic. But in refactor step 4 - "refactor accelerator_connector", we will revisit this part of the logic.